litmus-portal/frontend/src/pages/GetStartedPage/index.tsx

```diff
@@ -190,8 +216,10 @@ const GetStarted: React.FC = () => {
                   title="Skip for now"
                   variant="highlight"
                   onClick={() => {
-                    setIsLoading(true);
-                    getUserInfo();
+                    if (ValidateUser()) {
```

Comment:
PTAL on above comment. Also If it is needed here then, it should be added in onClick() of Submit button.

Reply:
State changes on every input in the input field. But if a user directly clicks on skip button, then we should have a check to prevent any other API calls. That's why it's added here